comm/src/main/scala/coop/rchain/comm/transport/TcpTransportLayer.scala

```diff
@@ -211,7 +211,7 @@ class TcpTransportLayer(host: String, port: Int, cert: String, key: String, maxM
     Task.delay {
       new TcpServerObservable(port, serverSslContext, maxMessageSize)
         .mapParallelUnordered(parallelism)(dispatchInternal)
-        .subscribe()(Scheduler.computation(parallelism, "tl-dispatcher"))
+        .subscribe()(Scheduler.fixedPool("tl-dispatcher", parallelism))
```

Comment:
Why did you choose `fixedPool` ? Isn't the TCP layer non-blocking? If so then why you didn't choose `Scheduler.io` ? Why not _cached_ thread pool?

Reply:
First of all, this is the thread pool for consuming the message queue and dispatching the messages to Casper and Connect. There shouldn't be any IO here and I expect the work to be CPU-bound. On the other hand, a `Scheduler.compution` thread pool is backed by a `ForkJoinPool` which potentially creates and removes threads. We need a fixed size thread pool with `Math.max(Runtime.getRuntime.availableProcessors(), 2)` thread. Maybe we can at some point in future use the global `node-runner` thread pool.